google/cloud/bigtable/instance.py

```diff
@@ -641,6 +644,184 @@ def list_tables(self):
+                		include AND, OR, and NOT expressions explicitly.
+
+                		Some examples of using filters are:
+
+                -  ``name:"exact"`` --> The Backup name is the string "exact".
+                -  ``name:howl`` --> The Backup name contains the string "howl"
+                -  ``source_table:prod`` --> The source table's name contains
+                		the string "prod".
+                -  ``state:CREATING`` --> The Backup is pending creation.
+                -  ``state:READY`` --> The Backup is created and ready for use.
+                -  ``(name:howl) AND (start_time < \"2020-05-28T14:50:00Z\")``
+                		--> The Backup name contains the string "howl" and
+                		the Backup start time is before 2020-05-28T14:50:00Z.
+                -  ``size_bytes > 10000000000`` --> The Backup size is greater
+                		than 10GB
+
+        :type order_by: str
+        :param order_by: (Optional) An expression for specifying the sort order
+        				 of the results of the request. The string value should
+        				 specify one or more fields in ``Backup``. The full
+        				 syntax is described at https://aip.dev/132#ordering.
+
+                		 Fields supported are: \* name \* source\_table \*
+                		 expire\_time \* start\_time \* end\_time \*
+                		 size\_bytes \* state
+
+                		 For example, "start\_time". The default sorting order
+                		 is ascending. To specify descending order for the
+                		 field, a suffix " desc" should be appended to the
+                		 field name. For example, "start\_time desc". Redundant
+                		 space characters in the syntax are insigificant. If
+                		 order_by is empty, results will be sorted by
+                		 ``start_time`` in descending order starting from
+                		 the most recently created backup.
+
+        :type page_size: int
+        :param page_size: (Optional) The maximum number of resources contained
+        				  in the underlying API response. If page streaming is
+        				  performed per-resource, this parameter does not
+        				  affect the return value. If page streaming is
+        				  performed per-page, this determines the maximum
+        				  number of resources in a page.
+
+        :rtype: :class:`~google.api_core.page_iterator.Iterator`
+        :returns: Iterator of :class:`~google.cloud.bigtable.backup.Backup`
+                  resources within the current Instance.
+        :raises: :class:`ValueError <exceptions.ValueError>` if one of the
+                 returned Backups' name is not of the expected format.
+        """
+        cluster_id = cluster_id or "-"
+
+        backup_list_pb = self._client.table_admin_client.list_backups(
+            self.name + "/clusters/" + cluster_id,
+            filter_=filter_,
+            order_by=order_by,
+            page_size=page_size,
```

Comment:
According to `bigtable_table_admin.pb2.py`,
```
      page_size:
          Number of backups to be returned in the response. If 0 or
          less, defaults to the server’s maximum allowed page size.
```
In the `list_backups` method of `bigtable_table_admin_client.py`, this parameter defaults to `None`, so here we follow the same logic. But I agree, it should probably default to `0` rather than none-type, will change.